src/meta/processors/BaseProcessor.inl

```diff
@@ -126,6 +127,44 @@ StatusOr<std::vector<nebula::cpp2::HostAddr>> BaseProcessor<RESP>::allHosts() {
     return hosts;
 }
 
+template<typename RESP>
+StatusOr<std::vector<cpp2::HostItem>> BaseProcessor<RESP>::allHostsWithStatus() {
+    std::vector<cpp2::HostItem> hostItems;
+    const auto& prefix = MetaServiceUtils::hostPrefix();
+    std::unique_ptr<kvstore::KVIterator> iter;
+    auto ret = kvstore_->prefix(kDefaultSpaceId_, kDefaultPartId_, prefix, &iter);
+    if (ret != kvstore::ResultCode::SUCCEEDED) {
```

Comment:
If whitelist disabled,  no hosts existed in rocksdb is normal.

Reply:
nice catch